templates/heat/bin/init.sh

```diff
@@ -45,7 +45,7 @@ done
 # and have it automatically detected, or would we have to somehow change the call
 # to the heat binary to tell it to use that custom conf dir?
 echo merging /var/lib/config-data/default/custom.conf into ${SVC_CFG_MERGED}
-crudini --merge ${SVC_CFG_MERGED} </var/lib/config-data/default/custom.conf
+crudini --merge ${SVC_CFG_MERGED} </var/lib/config-data/default/01-custom.conf
```

Comment:
one interesting thing here (maybe for a follow up) would be to remove `crudini` and remove (entirely) the `initscript`.
- `00-default.conf` can be a static file, copied by kolla during `kolla_start` (see what I did here [1] [2]).
This way you can simplify a lot the startup logic and let kolla do both copy and merge for you in a `heat.conf.d` directory where your 00, 01, 02, 03 configs are stored.

[1] https://github.com/openstack-k8s-operators/glance-operator/blob/main/templates/common/config/00-config.conf
[2] https://github.com/openstack-k8s-operators/glance-operator/blob/main/templates/glanceapi/config/glance-api-config.json

Reply:
oh that's good, works for me then, I'm +1 to focus PRs on a smaller subset of changes.